src/CattyTests/PlayerEngine/AudioEngine/Mocks/AudioEngineFingerprintingStub.swift

```diff
@@ -36,7 +36,10 @@ class AudioEngineFingerprintingStub: Pocket_Code.AudioEngine {
             speechSynth.utteranceVolume = muted ? 0.0 : 1.0
         }
 
-        recorder = try? AudioKit.NodeRecorder(node: engineOutputMixer)
+        let tempFile: AVAudioFile? = AudioKit.NodeRecorder.createTempFile()
+        let file = try? AVAudioFile(forWriting: tempFile!.url, settings: tempFile!.fileFormat.settings)
+
+        recorder = try? AudioKit.NodeRecorder(node: engineOutputMixer, file: file)
```

Comment:
Are this changes still necessary with AK 5.3.1? Should be done automatically

Reply:
Yes it is necessary, otherwise the test cases fail.